forte/sci/fci_mo.h

```diff
@@ -264,6 +267,19 @@ class FCI_MO : public ActiveSpaceMethod {
     /// Return the parsed state-averaged info
     std::vector<std::tuple<int, int, int, std::vector<double>>> sa_info() { return sa_info_; }
 
+    /// Return 1 coupling coefficients
+    std::vector<ambit::Tensor> coupling_coeffcients_1();
```

Comment:
This and the below two functions are `const`, no?

Reply:
Yes, I have added `const` for all.